### PR TITLE
fast debug drawing and capsule height and width fix

### PR DESCRIPTION
**Pull request by jordo**
This has faster debug drawing which improves performance significantly when is_debugging_collisions_hint is turned on.

Also includes capsule fixture fix.

### CODE REVIEW

**Comment by jordo** [comment]
Just note on this PR, this is kinda clobbering this:  (reverted some things).  

```
			if (body && is_contact_monitor_enabled()) {
				for (int i = 0; i < get_contact_count(); i++) {
					Vector2 point = get_box2dworld_transform().xform_inv(get_contact_world_pos(i));
					draw_rect(Rect2(point + Point2(-1.0f, -1.0f), Size2(2.0f, 2.0f)), Color(1.0f, 1.0f, 0.0f));
```

**Comment by jordo** [comment]
Up for discussion how to do this, but this looks like we just want to show contacts as well for each body?

**Comment by briansemrau** [comment]
I hadn't really decided whether we should or how to go about drawing debug contacts. In the future I think they should be made useful for debugging advanced collision logic.

File: scene/2d/box2d_fixtures.cpp [hunk 1]
```diff
@@ -144,42 +144,35 @@ void Box2DFixture::_notification(int p_what) {
 
 				body_node = new_body;
```
Comment (briansemrau) [inline scene/2d/box2d_fixtures.cpp:149]: We should disconnect signals before we reassign body_node. (if body_node != null)

**Review by briansemrau — CHANGES_REQUESTED** [review]

**Review by briansemrau — APPROVED** [review]

**Pull request merged**